src/Tools/ExternalAccess/OmniSharp/Internal/ExtractClass/OmniSharpExtractClassOptionsService.cs

```diff
@@ -25,9 +27,9 @@ public OmniSharpExtractClassOptionsService(IOmniSharpExtractClassOptionsService
             _omniSharpExtractClassOptionsService = omniSharpExtractClassOptionsService;
         }
 
-        public async Task<ExtractClassOptions?> GetExtractClassOptionsAsync(Document document, INamedTypeSymbol originalType, ISymbol? selectedMember, CancellationToken cancellationToken)
+        public async Task<ExtractClassOptions?> GetExtractClassOptionsAsync(Document document, INamedTypeSymbol originalType, ImmutableArray<ISymbol> selectedMembers, CancellationToken cancellationToken)
```

Comment:
I'm not sure why this is changed if it's not actually going to use anything more than the first item?

Reply:
`IOmniSharpExtractClassOptionsService` is an internal API. The contract is that O# needs to react when we update it, and either Joey or myself signs off to indicate that we've seen the notification of the breaking change.

Reply:
Ok, I'll make the change to the interface as well then. Would there need to be anything done to notify/check with O# before that change is made?

Reply:
> Would there need to be anything done to notify/check with O# before that change is made?

That's just notifying @JoeRobich and myself :).